eva_sub_cli/jinja_templates/sample_name_check.html

```diff
@@ -1,6 +1,10 @@
 
 {% macro sample_name_check_report(validation_results) -%}
     {% set results = validation_results.get('sample_check', {}) %}
+    {% macro format_error(error) %}
+        {{ error | replace(' ', '•') }}
```

Comment:
What about other types of whitespace (tabs and maybe newlines)? I guess they're very unlikely particularly in spreadsheets, but may be possible in JSON.

Reply:
I kind of agree but I also think it might get complicated to display tab and newline as their specific character.
I would suggest that we think of a more generalised solution (unicode maybe) if that becomes a problem.